### PR TITLE
Fix runner bugs

### PR DESCRIPTION
**Pull request by no-longer-on-githu-b**

### CODE REVIEW

File: runapl [hunk 1]
```diff
@@ -7,14 +7,16 @@ File.exists?(file || "") || (puts "invalid program '#{file}'"; exit 1)
...
-# - f(ile) to load from a file
-system "apl --noSV -s -f #{temp.path}"
-
-temp.unlink
+  # run apl
+  # - noSV to disable the shared variable server
+  # - s (better --silent) to disable banner etc
+  # - f(ile) to load from a file
+  system "apl", "--noSV", "-s", "-f", temp.path
+ensure
```
Comment (vendethiel) [inline runapl:20]: this is very MEH in ruby, since the GC and everything else will ensure the tempfile is deleted

Comment (no-longer-on-githu-b) [inline runapl:20]: not guaranteed by ISO standard

Comment (no-longer-on-githu-b) [inline runapl:20]: but at least merge the second commit :p

**Comment by vendethiel** [comment]
guess it's good anyway

**Pull request merged**